angular/wolox-books/src/app/screens/register/register.component.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../scss/colors";
+
+.form-container {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  height: 100%;
+
+  .register-form {
+    background-color: $wild-sand;
+    border-top: 4px solid $blue;
+    display: flex;
+    flex-direction: column;
+    max-width: 350px;
+    padding: 30px 20px 20px;
+    width: 100%;
+
+    .logo {
+      align-self: center;
+      margin-bottom: 20px;
+      max-height: 60px;
+    }
+
+    .label {
+      line-height: 2;
+      margin-bottom: 10px;
+    }
+
+    .input {
+      border-radius: 7px;
+      display: block;
+      padding: 10px;
+      width: 100%;
+
+      &.ng-invalid.ng-touched,
+      &.invalid {
+        border: 1px solid $red;
+      }
+    }
+
+    .btn {
```

Comment:
I'd move the buttons to a generic `components.scss` file since they might want to be reused elsewhere